app/src/Components/Error/UnauthorisedError.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './../../styles/baseStyles';
+import LinkButton from './../FormInput/LinkButton';
+
+export const UnauthorisedError = () => <div style={{ 'display': 'flex', 'justifyContent': 'center', 'alignItems': 'center' }}>
```

Comment:
Do we use British or American English in our code?

Reply:
whooops..